.kitchen.docker.yml

```diff
@@ -29,4 +29,5 @@ suites:
       inspec_tests:
         - name: chef-aps-inspec-tomcat
           git: https://github.com/Alfresco/chef-aps-inspec-tomcat
+          branch: 'TAA-930'
```

Comment:
We should not merge in develop with a specific branch, let's wait to merge chef-aps-inspec-tomcat into master